inspirehep/modules/theme/templates/inspirehep_theme/format/record/Journal_HTML_detailed.tpl

```diff
@@ -47,13 +48,14 @@
           <div class="col-md-12">
             {% if record.publisher %}
               <div class="detailed-record-field">
-              <label>Published by:</label>
-              {{ record.publisher }}
-              <br>
+                <label>Published by:</label>
+                {{ record.publisher }}
```

Comment:
`publisher` is an array in the schema http://inspire-schemas.readthedocs.io/en/latest/schemas/records/journals.html#publisher so this will not work correctly.

You can get some examples from https://inspirehep.net/search?ln=en&cc=Journals&p=643__b%3A%2F.*%2F&action_search=Search

Go into any of the results, get the ID from the URL and load it locally with:

```
docker-compose run --rm web inspirehep migrator one --recid=1617783
```

Reply:
Or even better load all the journals from http://inspirehep.net/dumps/inspire-dump.html and you will have more examples

Reply:
Ah. Okay